cli/src/main/java/io/aklivity/zillabase/cli/internal/migrations/comparator/ZillabaseSchemaComparator.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2024 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zillabase.cli.internal.migrations.comparator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.aklivity.zillabase.cli.internal.migrations.model.ZillabaseDatabaseSchema;
+import io.aklivity.zillabase.cli.internal.migrations.model.ZillabaseSchemaDiff;
+
+public final class ZillabaseSchemaComparator
+{
+    @FunctionalInterface
+    private interface SchemaComparisonStrategy
+    {
+        void compare(
+            ZillabaseDatabaseSchema actual,
+            ZillabaseDatabaseSchema expected,
```

Comment:
Note: `expected` usually comes before `actual` in a straight comparison like this.
Is this really `expected` and `actual` though? or is it `from` and `to`?

Reply:
I think it is `from` and `to`